clients/unshaded/src/main/java/tachyon/client/table/RawTable.java

```diff
@@ -15,91 +15,41 @@
 
 package tachyon.client.table;
 
-import java.io.IOException;
-import java.nio.ByteBuffer;
-
-import com.google.common.base.Preconditions;
-
-import tachyon.client.TachyonFS;
-import tachyon.thrift.RawTableInfo;
-import tachyon.util.io.BufferUtils;
+import tachyon.annotation.PublicApi;
 
 /**
  * Tachyon provides native support for tables with multiple columns. Each table contains one or more
- * columns. Each columns contains one or more ordered files.
+ * columns. Each column contains one or more ordered files.
  */
+@PublicApi
 public class RawTable {
-  private final TachyonFS mTachyonFS;
-  private final RawTableInfo mRawTableInfo;
 
-  /**
-   * Creates a new <code>RawTable</code>
-   *
-   * @param tachyonClient the <code>TachyonFS</code> client
-   * @param rawTableInfo information describing the table
-   */
-  public RawTable(TachyonFS tachyonClient, RawTableInfo rawTableInfo) {
-    mTachyonFS = tachyonClient;
-    mRawTableInfo = rawTableInfo;
-  }
+  /** Id of the raw table, which uniquely identifies this table */
+  private final long mRawTableId;
 
   /**
-   * @return the number of columns of the raw table
+   * Creates a raw table which is used as a handler for accessing raw tables in
+   * {@link TachyonRawTables}
+   *
+   * @param rawTableId the id of the raw table
    */
-  public int getColumns() {
-    return mRawTableInfo.getColumns();
+  public RawTable(long rawTableId) {
+    mRawTableId = rawTableId;
   }
 
   /**
    * @return the id of the raw table
    */
-  public long getId() {
-    return mRawTableInfo.getId();
-  }
-
-  /**
-   * @return the meta data of the raw table
-   */
-  public ByteBuffer getMetadata() {
-    return BufferUtils.cloneByteBuffer(mRawTableInfo.metadata);
-  }
-
-  /**
-   * @return the name of the raw table
-   */
-  public String getName() {
-    return mRawTableInfo.getName();
+  public long getRawTableId() {
+    return mRawTableId;
   }
 
   /**
-   * @return the path of the raw table
-   */
-  public String getPath() {
-    return mRawTableInfo.getPath();
-  }
-
-  /**
-   * Get one column of the raw table.
-   *
-   * @param columnIndex the index of the column
-   * @return the RawColumn
-   */
-  public RawColumn getRawColumn(int columnIndex) {
-    Preconditions.checkArgument(columnIndex >= 0 && columnIndex < mRawTableInfo.getColumns(),
-        mRawTableInfo.getPath() + " does not have column " + columnIndex + ". It has "
-            + mRawTableInfo.getColumns() + " columns.");
-
-    return new RawColumn(mTachyonFS, this, columnIndex);
-  }
-
-  /**
-   * Update the meta data of the raw table
-   *
-   * @param metadata the new meta data
-   * @throws IOException if an event that prevents the metadata from being updated is encountered
+   * @param columnIndex the index of the column to fetch
+   * @return a handler for a column of this raw table, the column index is not checked, if the
```

Comment:
would the part after "@return a handler for a column of this raw table" make more sense outside of the `@return`?

Also, should handler be handle?